packages/manager/modules/nasha/src/components/partition/zfs-options/zfs-options.controller.js

```diff
@@ -66,6 +68,11 @@ export default class NashaComponentsPartitionZfsOptionsController {
       .catch((error) => this.close({ error }));
   }
 
+  onClickOnClose() {
```

Comment:
Same as https://github.com/ovh/manager/pull/7762/files#r943141172

Reply:
It's applied.